packages/strapi/lib/commands/develop.js

```diff
@@ -93,7 +94,24 @@ module.exports = async function({ build, watchAdmin, browser }) {
         }
       });
 
-      return strapiInstance.start();
+      
+      if (!repl) {
+        return strapiInstance.start();
+      } else {
+        return strapiInstance.start(() => {
+          const repl_ = REPL.start(strapiInstance.config.info.name + ' > ' || 'strapi > '); // eslint-disable-line prefer-template
```

Comment:
this will display "undefiend >" if the config is not set so the || is never used. 
please use a tempalte string and remove the eslint-disable